src/App.js

```diff
@@ -0,0 +1,227 @@
+import React, {Component} from 'react';
+import FormContainer from './containers/FormContainer/FormContainer';
+import {BrowserRouter} from 'react-router-dom';
+
+class App extends Component {
+    constructor() {
+        super();
+        this.state = {
+            todos: [
+                // {id:1 content:"Task1", done:false}
+            ]
+        }
+
+
+    }
+
+    // fetching todos from db.json
+    fetchData = (url) => {
+        fetch(url)
+            .then((response) => {
+                console.log(response);
+                return response
+            })
+            .then((response) => response.json())
+            .then((todos) => this.setState({todos: todos}))
+            .catch((error) => console.error(error));
+    };
+
+    //GET TODOS
+
+    fetchDataHandler = () => {
+        this.fetchData('http://localhost:3000/todos')
+    };
+
+
+    componentWillMount = () => {
+        this.fetchData('http://localhost:3000/todos');
+        console.log('component will mount');
+        console.log(this.state.todos);
+    };
+
+
+    // these method belows are used for CRUD of todos on json-server, not state.
+
+    //POST
+
+    postTodoHandler = (content_input) => {
```

Comment:
you should use camelCase for everything, for consistency.

Reply:
ahh, small mistake but I will be careful!